.github/workflows/test-e2e-junk.yaml

```diff
@@ -0,0 +1,75 @@
+name: TestSuite E2E Junk
+
+on:
+  pull_request:
+    branches:
+      - "main"
+  workflow_dispatch:
+    inputs:
+      k8s_version:
+        description: With Kubernetes version
+        required: false
+
+jobs:
+  wait-for-image-build:
+    name: Wait for image build
+    runs-on: ubuntu-latest
+    steps:
+      - uses: kyma-project/lifecycle-manager/.github/actions/wait-for-image-build@main
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          statusName: pull-runtime-watcher-img-build
+
+  e2e-integration:
+    name: E2E
+    needs: wait-for-image-build
+    strategy:
+      matrix:
+        e2e-test:
+          - watcher-enqueue
+          - watcher-metrics
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    steps:
+      - name: Checkout runtime-watcher
+        uses: actions/checkout@v4
+
+      - name: Checkout lifecycle-manager
+        uses: actions/checkout@v4
+        with:
+          repository: kyma-project/lifecycle-manager
+          path: lifecycle-manager
```

Comment:
comment: previously we also checked out template operator https://github.com/kyma-project/runtime-watcher/blob/main/.github/workflows/test-e2e-runtime-watcher.yml#L116-L120

But it appeared unnecessary so I removed it. Tests are still green.